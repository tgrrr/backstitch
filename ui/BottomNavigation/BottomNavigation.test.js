import React from 'react'
import renderer from 'react-test-renderer'
import { BottomNavigation } from '.'

it('BottomNavigation: default', () => {
    const component = renderer.create(
        <BottomNavigation>
            <span>
                BottomNavigation
            </span>
        </BottomNavigation>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
