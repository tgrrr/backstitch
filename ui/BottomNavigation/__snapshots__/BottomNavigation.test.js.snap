// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomNavigation: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Bottomnavigation "
    data-testid="Bottomnavigation"
  >
    <div
      className="MuiBottomNavigation-root"
    >
      BottomNavigation
    </div>
  </div>
</div>
`;
